### ISSUE TITLE
About pretraining steps

### ISSUE THREAD

**Issue by reflectionie** [zzz47zzz/codebase-for-incremental-learning-with-llm#1]
Hi, thanks for your work. May I ask what the definition of 'pretraining step' is in your paper? As far as I understand, you did not train the LLM from scratch. So, does 'pretraining' in the 'pretraining step' you mentioned actually refer to fine-tuning on the 'Concept-1K' dataset? Additionally, do the 'pretraining steps' in your paper refer to the number of gradient updates or the number of incremental steps?

**Comment by zzz47zzz** [comment]
Thank you for your attention to our paper! We use [Pythia](https://github.com/EleutherAI/pythia) pretrained models in our paper. Therefore, we do not need to pretrain a large language model (LLM) from scratch. Instead, we load the checkpoint from Huggingface's Pythia directly. For example, we can load the Pythia-70m with pretraining steps set to 3000 using the following code:
```
model = GPTNeoXForCausalLM.from_pretrained(
  "EleutherAI/pythia-70m-deduped",
  revision="step3000",
  cache_dir="./pythia-70m-deduped/step3000",
)
```
For more details, you can refer to the [Pythia documentation](https://github.com/EleutherAI/pythia).
